packages/turbo-vsc/package.json

```
@@ -1,8 +1,8 @@
 {
   "name": "turbo-vsc",
-  "displayName": "Turborepo",
+  "displayName": "Turbo LSP",
```

```suggestion
  "displayName": "Turborepo LSP",
```

We only use Turborepo, or `turbo`. `turbo` refers to the executable directly, and Turborepo to the product.